packages/main/src/Input.js

```diff
@@ -581,17 +585,38 @@ class Input extends UI5Element {
 	}
 
 	get ariaDescribedBy() {
+		if (this._accInfo && this._accInfo.ariaDescribedBy) {
+			return `${this.suggestionsTextId} ${this.valueStateTextId} ${this._accInfo.ariaDescribedBy}`.trim();
+		}
 		return `${this.suggestionsTextId} ${this.valueStateTextId}`.trim();
 	}
 
 	get ariaHasPopup() {
+		if (this._accInfo && this._accInfo.ariaHasPopup) {
+			return this._accInfo.ariaHasPopup;
+		}
 		return this.showSuggestions ? "true" : undefined;
 	}
 
 	get ariaAutoComplete() {
+		if (this._accInfo && this._accInfo.ariaAutoComplete) {
```

Comment:
Same as above, you can do one check less, as undefined is fine:

if (this._accInfo) {
return this._accInfo. ariaAutoComplete;
}